src/controllers/generalSettings.js

```diff
@@ -0,0 +1,7 @@
+const fixedVariables = require('../database/models/fixedVarible');
+
+exports.get = (req, res) => {
+  fixedVariables.findAll({ attributes: ['dolar', ['exemption_resident', 'exemptionResident'], ['tax_value', 'taxValue'], ['contribution_institute', 'contributionInstitute'], ['contribution_employee', 'contributionEmployee'], ['saving_ratio', 'savingRatio']] }).then((result) => {
```

Comment:
Avoid having lines of code that are longer than 100 characters (including whitespace).

[more info](https://github.com/airbnb/javascript#whitespace--max-len)